dipy/reconst/shm.py

```diff
@@ -1575,3 +1575,52 @@ def convert_sh_to_legacy(sh_coeffs, sh_basis, full_basis=False):
         raise ValueError("Invalid basis name.")
 
     return out_sh_coeffs
+
+
+def convert_sh_descoteaux_tournier(sh_coeffs):
+    """Convert SH coefficients between the legacy ``descoteaux07`` SH basis and
+    the non-legacy ``tournier07`` SH basis. Because this conversion is equal to
+    its own inverse, it can be used to convert in either direction:
+    legacy-descoteaux to non-legacy-tournier or non-legacy-tournier to
+    legacy-descoteaux.
```

Comment:
Can you update the docstring to respect our standard. see [here](https://github.com/dipy/dipy/blob/master/dipy/direction/peaks.py#L94-L131) for an example

Reply:
Can you be more specific about what needs to be changed? It seems to follow the example already.

Reply:
Sorry, the first summary should be one-line summary. 

So, it would be great to shorten this sentence `Convert SH coefficients between the legacy ``descoteaux07`` SH basis and the non-legacy ``tournier07`` SH basis.`

and move the sentence starting with `because this conversion ....` to the extended summary (so in a new line, separated by an empty line.)

we follow Numpy  docstring standard: https://numpydoc.readthedocs.io/en/latest/format.html

Thank you for the future update